src/Benchmarks/Benchmarks.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>GraphQL.Server.Benchmarks</RootNamespace>
+    <IsPackable>false</IsPackable>
```

Comment:
```suggestion
```

Reply:
I did this as I saw it was there in the graphql-dotnet Benchmarks project. Makes sense to never let it be packable.

Reply:
graphql-dotnet has a bit different build settings - graphql-dotnet/graphql-dotnet#1481 , it packs all that is possible so we need to opt-out `IsPackable` explicitly.